packages/map/src/CdcMap.jsx

```diff
@@ -177,8 +177,8 @@ const CdcMap = ({ className, config, navigationHandler: customNavigationHandler,
 
   // Tag each row with a UID. Helps with filtering/placing geos. Not enumerable so doesn't show up in loops/console logs except when directly addressed ex row.uid
   // We are mutating state in place here (depending on where called) - but it's okay, this isn't used for rerender
+  // eslint-disable-next-line
   const addUIDs = useCallback((obj, fromColumn) => {
-    // eslint-disable-line
```

Comment:
@tturnerswdev33 - I updated this. `// eslint-disable-line` here, the way you had it, would do nothing. Added the eslint ignore above the function as `// eslint-disable-next-line`